RefactoringToDesignPatterns/StrategyPatternPractice/Task/test/Tests.kt

```diff
@@ -91,21 +103,21 @@ class StrategyPatternTest : BaseIjTestClass() {
     fun testInvokedMethodsOfPaymentClassesInMainMethod() {
         setUp()
         myFixture.configureByText("Main.kt", sourceText)
-        var method = "processOrderPayment(100.0)"
+        var method = "processOrderPayment(order1.totalAmount)"
         Assertions.assertEquals(
             findMethodUsages(method),
             listOf("main")
         ) {
             "Please, invoke the $method method of Credit Card Payment in the main method"
```

Comment:
Could you please fix the error messages? We need to not only invoke the method but also pass the corresponding order to it

Reply:
thanks! done